standup_notes/__main__.py

```diff
@@ -1,33 +1,35 @@
+# PYTHON_ARGCOMPLETE_OK
 import os
 import sys
-from argparse import ArgumentParser, Namespace
 from datetime import date, timedelta
 from typing import Callable
 
-from pkg_resources import resource_stream
-
+import argcomplete
```

Comment:
Isn't argcomplete supposed to set up the bash completion for you instead of you having to define the completion script yourself? I could be wrong but that was what I thought. If I'm not mistaken though, it seems like now that you've defined our own bash completion script, we can just remove the dependency on argcomplete and do bash completion ourselves.

Reply:
Since there is the bash completion there is no need for it anymore.